electrometer/docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version : "3.6"
+services:
+    electrometer:
+        image: lsstts/electrometer:develop
+        container_name: electrometer
+        devices:
+            - "/dev/ttyUSB0:/dev/electrometer"
```

Comment:
This can become messy at times. Kernel doesn't guarantee /dev/ttyUSB0 will be the USB-serial. I highly recommend to add to dependencies options how to create a [simlink for give ttyUSB](https://rts2.org/wiki/doku.php?id=howto:set_symlinks_for_usb_serial_ports) and use this simlink on host (so mapping would be e.g. /dev/electrometer:/dev/elmeter)